tests/flows/erc20_source_erc20_destination.go

```diff
@@ -163,3 +163,158 @@ func ERC20SourceERC20Destination(network interfaces.Network) {
 	Expect(err).Should(BeNil())
 	Expect(balance).Should(Equal(bridgedAmount))
 }
+
+/**
+ * Deploy an ERC20 token source on the primary network
+ * Defers deploying ERC20Destination to Subnet A
+ * Bridges C-Chain example ERC20 tokens to Subnet A
+ * Confirms failure to execute message, since destination hasn't been deployed.
+ * Deploys ERC20Destination to Subnet A
+ * Invokes a retry of message execution
+ * Confirms message execution
+ */
+func ERC20DestinationNotYetDeployed(network interfaces.Network) {
+	cChainInfo := network.GetPrimaryNetworkInfo()
+	subnetAInfo, _ := teleporterUtils.GetTwoSubnets(network)
+	fundedAddress, fundedKey := network.GetFundedAccountInfo()
+
+	ctx := context.Background()
+
+	// Deploy an ExampleERC20 on the primary network as the source token to be bridged
+	sourceTokenAddress, sourceToken := teleporterUtils.DeployExampleERC20(
+		ctx,
+		fundedKey,
+		cChainInfo,
+	)
+
+	// Create an ERC20Source for bridging the source token
+	erc20SourceAddress, erc20Source := utils.DeployERC20Source(
+		ctx,
+		fundedKey,
+		cChainInfo,
+		fundedAddress,
+		sourceTokenAddress,
+	)
+
+	// Token representation on subnet A will have same name, symbol, and decimals
+	tokenName, err := sourceToken.Name(&bind.CallOpts{})
+	Expect(err).Should(BeNil())
+	tokenSymbol, err := sourceToken.Symbol(&bind.CallOpts{})
+	Expect(err).Should(BeNil())
+	tokenDecimals, err := sourceToken.Decimals(&bind.CallOpts{})
+	Expect(err).Should(BeNil())
+
+	// pre-determine the deployment address of the ERC20Destination, but
+	// don't deploy it yet
+	erc20DestinationAddressNonce, err := subnetAInfo.RPCClient.NonceAt(
+		ctx,
+		fundedAddress,
+		nil,
+	)
+	Expect(err).Should(BeNil())
+	erc20DestinationAddressNonce += 1
+	erc20DestinationAddress := crypto.CreateAddress(
+		fundedAddress,
+		erc20DestinationAddressNonce,
+	)
+	erc20Destination, err := erc20destination.NewERC20Destination(
```

Comment:
we don't need this call, when we `DeployERC20Destination` it returns an `erc20Destination`.

Reply:
addressed in [9238bb6](https://github.com/ava-labs/teleporter-token-bridge/pull/85/commits/9238bb6c7e9e43e5d1ab3181091a86993d1a427c)